src/yoti_request/aml.result.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+// AML result attributes
```

Comment:
ditto

Reply:
I also prefer to keep those comments there - might not be apparent for some people.